t/test.py

```diff
@@ -126,11 +126,6 @@ def test_subtract_time_delta(self):
         self.assertEqual(new_date.month, 4)
         self.assertEqual(new_date.day, 21)
         self.assertEqual(new_date.locale, 'nl_NL')
-    
-    def test_subtract_datetime_date(self):
```

Comment:
I'm curious why a formatting commit has removed this test case. could you please elaborate on this @hramezani?

Reply:
It was a duplicate test